pkg/providers/imagefamily/bootstrap/cse_cmd_nbcontract.sh.gtpl

```diff
@@ -65,23 +65,23 @@ TELEPORT_ENABLED="{{getBoolFromFeatureState .TeleportConfig.Status}}"
 TELEPORTD_PLUGIN_DOWNLOAD_URL={{.TeleportConfig.TeleportdPluginDownloadUrl}}
 RUNC_VERSION={{.RuncConfig.RuncVersion}}
 RUNC_PACKAGE_URL={{.RuncConfig.RuncPackageUrl}}
-ENABLE_HOSTS_CONFIG_AGENT="{{getBoolFromFeatureState .HostsConfigAgentStatus}}"
-DISABLE_SSH={{if .SshStatus}}"false"{{else}}"true"{{end}}
+ENABLE_HOSTS_CONFIG_AGENT="{{.GetEnableHostsConfigAgent}}"
+DISABLE_SSH="{{not .GetEnableSsh}}"
 SHOULD_CONFIGURE_HTTP_PROXY="{{getBoolStringFromFeatureStatePtr .HttpProxyConfig.Status}}"
 SHOULD_CONFIGURE_HTTP_PROXY_CA="{{getBoolStringFromFeatureStatePtr .HttpProxyConfig.CaStatus}}"
 HTTP_PROXY_TRUSTED_CA="{{.HttpProxyConfig.ProxyTrustedCa}}"
 HTTP_PROXY_URLS="{{.HttpProxyConfig.HttpProxy}}"
 HTTPS_PROXY_URLS="{{.HttpProxyConfig.HttpsProxy}}"
 NO_PROXY_URLS="{{getStringifiedStringArray .HttpProxyConfig.NoProxyEntries ","}}"
-SHOULD_CONFIGURE_CUSTOM_CA_TRUST="{{getBoolStringFromFeatureState .CustomCaTrustConfig.Status}}"
-CUSTOM_CA_TRUST_COUNT="{{len .CustomCaTrustConfig.CustomCaCerts}}"
-{{range $i, $cert := .CustomCaTrustConfig.CustomCaCerts}}
+SHOULD_CONFIGURE_CUSTOM_CA_TRUST="{{getCustomCACertsStatus .GetCustomCaCerts}}"
+CUSTOM_CA_TRUST_COUNT="{{len .GetCustomCaCerts}}"
+{{range $i, $cert := .CustomCaCerts}}
 CUSTOM_CA_CERT_{{$i}}="{{$cert}}"
 {{end}}
-IS_KRUSTLET="{{.IsKrustlet}}"
-IPV6_DUAL_STACK_ENABLED={{getBoolStringFromFeatureStatePtr .Ipv6DualStackEnabled}}
-ENABLE_UNATTENDED_UPGRADES={{getBoolStringFromFeatureStatePtr .UnattendedUpgradeStatus}}
-ENSURE_NO_DUPE_PROMISCUOUS_BRIDGE={{derefBool .EnsureNoDupePromiscuousBridge}}
+IS_KRUSTLET="{{getIsKrustlet .GetWorkloadRuntime}}"
+IPV6_DUAL_STACK_ENABLED="{{.GetIpv6DualStackEnabled}}"
+ENABLE_UNATTENDED_UPGRADES={{.GetEnableUnattendedUpgrade}}
+ENSURE_NO_DUPE_PROMISCUOUS_BRIDGE={{getEnsureNoDupePromiscuousBridge .GetNetworkConfig.GetNetworkPlugin .GetNetworkConfig.GetNetworkPolicy}}
```

Comment:
nit, but could also just pass in the network config and then access the fields in the function. Can be done later though, approving.

Reply:
I was thinking if for our go binary if it's easier for it pass the individual variable or the whole network config object. But I think the whole object is easier. I will follow your suggestion. Thanks :)